### ISSUE TITLE
babi-rnn example test accuracy jump to 0.2780 (from 0.2140)

### ISSUE THREAD

**Issue by obastemur** [keras-team/keras#13272]
Sample @ https://github.com/keras-team/keras/blob/master/examples/babi_rnn.py 

Updating the activation function from `softmax` to `relu` pushes accuracy to 0.2780 from 0.2140 and lowers test loss from 1.8003 to 1.7391. (not super surprising though)

Is there a particular reason that the sample is kept with `softmax` ?
